packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -268,7 +290,14 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	 */
 	public ngOnInit(): void {
 		this.logger.debug(componentName + ": component initialized");
-		this.displayedColumns = [];
+
+		// Emit event when selection changes
+		if (this.selection.changed) {
```

Comment:
Is this check really needed? Normally `this.selection` is initialized in its declaration and the "changes" observable should always be there right?

Reply:
Indeed, I don't remember why I did that. 🤔